sdks/python/apache_beam/runners/dataflow/dataflow_runner_test.py

```diff
@@ -176,6 +176,19 @@ def test_create_runner(self):
         isinstance(create_runner('TestDataflowRunner'),
                    TestDataflowRunner))
 
+  def test_environment_override_translation(self):
+    self.default_properties.append("--experiments=beam_fn_api")
+    self.default_properties.append("--worker_harness_container_image=FOO")
+    remote_runner = DataflowRunner()
+    p = Pipeline(remote_runner,
+                 options=PipelineOptions(self.default_properties))
+    (p | ptransform.Create([1, 2, 3])  # pylint: disable=expression-not-assigned
+     | 'Do' >> ptransform.FlatMap(lambda x: [(x, x)])
+     | ptransform.GroupByKey())
+    p.run()
+    pipeline_proto = remote_runner.pipeline_proto
+    self.assertEqual(pipeline_proto.components.envirnoments.values(), ["FOO"])
```

Comment:
Just to clarify, does "["FOO"]" somehow end up being equal to a DockerPayload with "FOO" set to container_image ?

Reply:
Consider adding a comment.

Reply:
It was a hastily made test, was relying on jenkins to test.